test-suite/vos/run.sh

```diff
@@ -1,13 +1,24 @@
 #!/bin/bash
 set -e
 set -o pipefail
+# export COQBIN=../../bin
```

Comment:
??

Reply:
This is a note for what one should to in order to run the script /run.sh  from the current folder.
Setting the path in the shell does not seem to work, because opam insists on its path to always appear to the front of $PATH.

Reply:
The customary way is `make -C test-suite foo/bar.log`, imo this test should go into misc